docs/src/lib.rs

```diff
@@ -19,7 +19,7 @@ use serde_yaml as yaml;
 use typst::doc::Frame;
 use typst::eval::{CastInfo, Func, FuncInfo, Library, Module, ParamInfo, Value};
 use typst::font::{Font, FontBook};
-use typst::geom::{Abs, Sides, Smart};
+use typst::geom::{Abs, Margin, Smart};
 use typst_library::layout::PageElem;
```

Comment:
(follow-up for my suggestion of moving `Margin` to `page.rs`)

```suggestion
use typst::geom::{Abs, Smart};
use typst_library::layout::{PageElem, Margin};
```

Reply:
Done.